Squest/settings.py

```diff
@@ -124,6 +136,25 @@
     },
 ]
 
+if SOCIAL_AUTH_OIDC_ENABLED and not TESTING:
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [TEMPLATES_DIR],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'social_django.context_processors.backends',
+                    'social_django.context_processors.login_redirect',
+                    'django.contrib.messages.context_processors.messages',
+                ],
+            },
+        },
+    ]
```

Comment:
Use this instead, it will avoid code duplication :
```py
TEMPLATES['OPTIONS']['context_processors'].append('social_django.context_processors.backends')
TEMPLATES['OPTIONS']['context_processors'].append('social_django.context_processors.login_redirect')
```

Reply:
This should be fixed now.